src/VisualStudio/Core/Impl/Options/AbstractSettingsManagerOptionSerializer.cs

```diff
@@ -27,16 +27,15 @@ private class SVsSettingsPersistenceManager { };
         protected readonly ISettingsManager Manager;
         private readonly IOptionService _optionService;
 
-        public AbstractSettingsManagerOptionSerializer(IServiceProvider serviceProvider, IOptionService optionService)
+        public AbstractSettingsManagerOptionSerializer(VisualStudioWorkspaceImpl workspace)
```

Comment:
I believe this has introduced a bug, since it means that we're now only raising the WorkspaceChanged event on the core VS workspace, and we're no longer raising it for MiscellaneousFilesWorkspaces.